modules/RTC/RTCBrowserType.js

```diff
@@ -163,11 +163,7 @@ const RTCBrowserType = {
     },
 
     usesPlanB() {
-        return (
-            RTCBrowserType.isChrome()
-                || RTCBrowserType.isOpera()
-                || RTCBrowserType.isReactNative()
-                || RTCBrowserType.isTemasysPluginUsed());
+        return !RTCBrowserType.usesUnifiedPlan();
```

Comment:
I like this! Actually now I'm think that before this changes maybe we had a bug if you are using Electron and NWJS. They should use Plan B right?